### DISCUSSION TITLE
How to Return cookie value from cookie_jar file as string

### DISCUSSION THREAD

**Discussion by MohdRashid01** [cookiecutter/cookiecutter#1603] [Q&A] [answered]
Hi All

I have one file that name is cookie_jar file in that file there is a cookie value stored in it that i need return in it as string way i have written one command but dont know how to put in as a bash script or shell script. So that my script should return as a token values. Here is the code

cookie_value = "'cookie: PF='$PF'; p_lplcwmsk='$p_lplcwmsk'; BIGipServerNP_QA_QFLOGINQAS0_7011='$BIGipServerNP_QA_QFLOGINQAS0_7011'; aatoken='$aatoken'; UserName='$UserName'; Auth='$Auth'; XSRF-TOKEN='$XSRF-TOKEN'; z='$z'; BN20='$BN20'; UserFirmId='$UserFirmId'; TS01a1bccb='$TS01a1bccb'; TS01e14722='$TS01e14722';"

echo cookie_value

please tell me how to write in script so that it can return string from cookie jar file as cookie value i want it that only from script

**Comment by luzfcb** [comment, marked as answer]
@MohdRashid01 your question is totally unrelated to what this project does. I think maybe you asked in the wrong repository.
To be clear, `cookiecutter` is a command-line utility that creates projects from [cookiecutters (project templates)](https://github.com/topics/cookiecutter-template), e.g. creating a Python package project from a Python package project template.

**Comment by MohdRashid01** [comment]
how to delete my question please tel me
